File: dia1/1876_Rabiola.py
try:
    while True:
        v = input()
        m = 0
        vs = v.split("x")
        m = max(len(vs[0]), len(vs[-1]))
        for f in vs[1:-1]:
            m = max(m, len(f) // 2)
        print(m)
except:
    pass

File: dia1/1876_Rabiola.cpp
#include <iostream>
using namespace std;

int main () {
	string str;
	while (cin >> str) {
		int cantX = 0, XVistas = 0, ultimaX;
		for (int i = 0; i < (int) str.size(); i++)
			if (str[i] == 'x')
				cantX++;		
	
		int res = 0;
		for (int i = 0; i < (int) str.size(); i++) {
			if (str[i] == 'x') {
				if (XVistas == 0) {
					res = i;
				}
				else {
					res = max (res, (i - ultimaX - 1) / 2);
				}

				XVistas++;
				ultimaX = i;
			}
		}
		res = max (res, (int) str.size() - ultimaX - 1);
		cout << res << endl;
	}
}
